pkg/storage/utitiltyStore.go

```diff
@@ -0,0 +1,24 @@
+package storage
+
+import (
+	_ "embed"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+)
+
+//go:embed SQL/utility/set_session_current_user.sql
+var setSessionCurrentUserSQL string
+
+// setCurrentSessionUserVariable sets the userID for
+func setCurrentSessionUserVariable(tx *sqlx.Tx, userID uuid.UUID) error {
+	argsUser := map[string]interface{}{
+		"user_id": userID.String(),
+	}
```

Comment:
Can this use the new `CreateUserIDQueryMap` function? I think they're the same thing here?

Reply:
sure! I changed it. I'm often on the fence about using query map helper functions. I do like the clarity of instantiating the map inline, but I think the name of the function should be sufficiently clear as well.